pkg/accounting/accounting.go

```diff
@@ -120,14 +125,22 @@ type Accounting struct {
 	// function used for time settlement
 	refreshFunction RefreshFunc
 	// allowance based on time used in pseudo settle
-	refreshRate *big.Int
+	refreshRate      *big.Int
+	lightRefreshRate *big.Int
 	// lower bound for the value of issued cheques
-	minimumPayment *big.Int
-	pricing        pricing.Interface
-	metrics        metrics
-	wg             sync.WaitGroup
-	p2p            p2p.Service
-	timeNow        func() time.Time
+	minimumPayment      *big.Int
+	pricing             pricing.Interface
+	metrics             metrics
+	wg                  sync.WaitGroup
+	p2p                 p2p.Service
+	timeNow             func() time.Time
+	thresholdGrowStep   *big.Int
+	thresholdGrowChange *big.Int
+	// light node counterparts
+	lightPaymentThreshold    *big.Int
```

Comment:
maybe not for this PR but it might be useful to have some kind of threshold "profile" struct, one for full nodes, one for light nodes. then each accounting peer can be given the appropriate one and just get stuff like `thresholdGrowChange` or the step from there. this may simplify the logic a bit and avoid some duplication.

Reply:
lets address this in a separate PR